.github/workflows/docker-ci.yml

```diff
@@ -0,0 +1,73 @@
+# This workflow uses actions that are not certified by GitHub.
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches: ["release"]
```

Comment:
We would like the Docker images to be updated with every push. We don't have a "release" branch yet. Replace it with the "master" branch.